package.json

```diff
@@ -1,93 +1,111 @@
 {
-  "name": "@storyblok/svelte-ws",
-  "private": true,
-  "version": "0.0.1",
-  "workspaces": [
-    "./lib",
-    "./playground",
-    "./playground-sveltekit"
+  "name": "@storyblok/svelte",
+  "type": "module",
+  "version": "5.0.0",
+  "private": false,
+  "packageManager": "pnpm@9.15.3",
+  "description": "SDK to integrate Storyblok into your project using Svelte.",
+  "author": "Storyblok",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/storyblok/storyblok-svelte.git"
+  },
+  "bugs": {
+    "url": "https://github.com/storyblok/storyblok-svelte/issues"
+  },
+  "sideEffects": [
+    "**/*.css"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js",
+      "import": "./dist/index.js",
+      "require": "./dist/index.js"
+    }
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "!dist/**/*.spec.*",
+    "!dist/**/*.test.*",
+    "dist"
   ],
-  "types": "./dist/types.index.d.ts",
+  "svelte": "./dist/index.js",
   "scripts": {
-    "dev": "npm run dev --workspace=lib & npm run demo",
-    "dev-kit": "npm run dev --workspace=lib & npm run demo-sveltekit",
-    "demo": "npm run demo --workspace=playground",
-    "demo-sveltekit": "npm run dev --workspace=playground-sveltekit",
-    "build": "npm run build --workspace=lib",
-    "build:ts": "vite build && tsc --prokect tsconfigjson",
-    "test:unit": "npm run test:unit --workspace=lib",
-    "test:e2e": "npm run test:e2e --workspace=lib",
-    "prepare": "husky install",
-    "check": "svelte-check --tsconfig lib/tsconfig.json"
+    "dev": "vite dev",
```

Comment:
@edodusi has the way to `dev` and `build` in Svelte changed? Asking as before, Svelte have a specific and recommended way to build package using `svelte-package` tailored for Svelte, as you can see here 
https://github.com/storyblok/storyblok-svelte/pull/1127/files#diff-1e946220773aef913945326261b7ee8d08b8ec29ccc66ef7c348950439212ffbL23-L24

Reply:
Actually just checked, and is still the recommended way in the current version of the docs https://svelte.dev/docs/kit/packaging. 

I'd recommend if possible, to use it across the different commands. Otherwise, the output from `build` and `package` can be different if we use different commands to build

Reply:
these scripts come straight from the Svelte official CLI (`pnpx sv@latest create`) so I can say this is the best practice.

`svelte-package` is actually used to build the library, see the `package` command below